packages/react/src/components/MultiSelect/MultiSelect.stories.js

```diff
@@ -21,6 +21,51 @@ export default {
       options: ['sm', 'md', 'lg'],
       control: { type: 'select' },
     },
+    downshiftProps: {
+      table: { disable: true },
+    },
+    compareItems: {
+      table: { disable: true },
+    },
+    sortItems: {
+      table: { disable: true },
+    },
+    initialSelectedItems: {
+      table: { disable: true },
+    },
+    id: {
+      table: { disable: true },
+    },
+    items: {
+      table: { disable: true },
+    },
+    local: {
+      table: { disable: true },
+    },
+    onChange: {
+      table: { disable: true },
+    },
+    onMenuChange: {
+      table: { disable: true },
+    },
+    itemToElement: {
+      table: { disable: true },
+    },
+    itemToString: {
+      table: { disable: true },
+    },
+    selectedItems: {
+      table: { disable: true },
+    },
+    open: {
+      table: { disable: true },
+    },
+    title: {
+      table: { disable: true },
+    },
+    translateWithId: {
+      table: { disable: true },
+    },
```

Comment:
```suggestion
    },
    light: {
      table: { disable: true },
    },
```
Could we add the light prop to this? I think the styles have been removed for it so the prop doesn't change anything